requirements.txt

```diff
@@ -22,5 +22,5 @@ PyJWT==1.6.4
 requests==2.20.0
 requests-http-signature==0.0.3
 rsa==4.0
-urllib3==1.23
+urllib3>=1.24.2, <1.25
```

Comment:
Do you see version ranges anywhere in this file? Stick with precedent.

Please pin to latest 1.24.x release.

Reply:
There are version ranges here:
https://github.com/DataBiosphere/azul/blob/0fd69736402df94283afb4d2f70eb69f2065cb12/requirements.txt#L10-L11